<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QVarNewdlg</class>
 <widget class="QDialog" name="QVarNewdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>4</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>变量</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>变量名称：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="m_varedit"/>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QComboBox" name="combo_datatype">
      <item>
       <property name="text">
        <string>位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16位整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32位整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16位正整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32位正整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16位BCD码</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32位BCD码</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32位浮点数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ASCII字符串</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="m_addredit"/>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="m_addrinput">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>数据类型：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>地 址：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="m_ok">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
